plugins/analytics-module-nr/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "backstage-plugin-analytics-module-nr",
```

Comment:
This name needs to be adjusted, it should be `@backstage/plugin-analytics-module-nr` and then that also needs to be updated in your changeset.

I would also like to suggest a larger name change here. I know we have used `ga` for the Google Analytics plugins but I wish we hadn't. I don't think using `nr` is clear and I think having it as `@backstage/plugin-analytics-module-newrelic-browser` would be much better, I'll leave this to @benjdlambert and/or @iamEAP to make the final call on though

Reply:
+1 here, think `newrelic` or `new-relic` should be in the name

Reply:
Yeah, I thought about that too. I'll change it to `@backstage/plugin-analytics-module-newrelic-browser`. Should I change this plugins folder name to `analytics-module-newrelic-browser` as well?

Reply:
Yep I would change the folder name to reflect the package name too.

Reply:
Made that change too.